service/src/main/java/bio/terra/workspace/common/utils/FlightUtils.java

```diff
@@ -79,6 +82,21 @@ public static void validateRequiredEntries(FlightMap flightMap, String... keys)
     }
   }
 
+  /**
+   * Validate that all common entries are present in the flight map
+   *
+   * @param flightMap input parameters
+   */
+  public static void validateCommonEntries(FlightMap flightMap) {
+    validateRequiredEntries(
```

Comment:
can we have a static field of COMMON_ENTRIES_LIST to be shared between the two methods?

Reply:
Good idea